### ISSUE TITLE
ItemJoin fails to load due to tab in plugin.yml

### ISSUE THREAD

**Issue by mibby** [RockinChaos/ItemJoin#233]
ItemJoin dev 299

@RockinChaos :( https://github.com/RockinChaos/ItemJoin/commit/d9e26ef808e61f7073a2a5019da09c3de6673503#diff-5df3d3f42757c737a95531b868d1d2b9R85

```
[02:41:17] [Server thread/ERROR]: Could not load 'plugins/ItemJoin-5.0.4-SNAPSHOT-b299.jar' in folder 'plugins'
org.bukkit.plugin.InvalidDescriptionException: Invalid plugin.yml
	at org.bukkit.plugin.java.JavaPluginLoader.getPluginDescription(JavaPluginLoader.java:169) ~[patched_1.14.4.jar:git-Paper-192]
	at org.bukkit.plugin.SimplePluginManager.loadPlugins(SimplePluginManager.java:142) ~[patched_1.14.4.jar:git-Paper-192]
	at org.bukkit.craftbukkit.v1_14_R1.CraftServer.loadPlugins(CraftServer.java:355) ~[patched_1.14.4.jar:git-Paper-192]
	at net.minecraft.server.v1_14_R1.DedicatedServer.init(DedicatedServer.java:226) ~[patched_1.14.4.jar:git-Paper-192]
	at net.minecraft.server.v1_14_R1.MinecraftServer.run(MinecraftServer.java:870) ~[patched_1.14.4.jar:git-Paper-192]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_222]
Caused by: org.yaml.snakeyaml.scanner.ScannerException: while scanning for the next token
found character '\t(TAB)' that cannot start any token. (Do not use \t(TAB) for indentation)
 in 'reader', line 85, column 1:
    			Example: /itemjoin menu - The ... 
    ^

	at org.yaml.snakeyaml.scanner.ScannerImpl.fetchMoreTokens(ScannerImpl.java:419) ~[patched_1.14.4.jar:git-Paper-192]
	at org.yaml.snakeyaml.scanner.ScannerImpl.checkToken(ScannerImpl.java:227) ~[patched_1.14.4.jar:git-Paper-192]
	at org.yaml.snakeyaml.parser.ParserImpl$ParseBlockMappingKey.produce(ParserImpl.java:558) ~[patched_1.14.4.jar:git-Paper-192]
	at org.yaml.snakeyaml.parser.ParserImpl.peekEvent(ParserImpl.java:158) ~[patched_1.14.4.jar:git-Paper-192]
	at org.yaml.snakeyaml.parser.ParserImpl.checkEvent(ParserImpl.java:148) ~[patched_1.14.4.jar:git-Paper-192]
	at org.yaml.snakeyaml.composer.Composer.composeMappingNode(Composer.java:214) ~[patched_1.14.4.jar:git-Paper-192]
	at org.yaml.snakeyaml.composer.Composer.composeNode(Composer.java:144) ~[patched_1.14.4.jar:git-Paper-192]
	at org.yaml.snakeyaml.composer.Composer.composeValueNode(Composer.java:236) ~[patched_1.14.4.jar:git-Paper-192]
	at org.yaml.snakeyaml.composer.Composer.composeMappingChildren(Composer.java:227) ~[patched_1.14.4.jar:git-Paper-192]
	at org.yaml.snakeyaml.composer.Composer.composeMappingNode(Composer.java:215) ~[patched_1.14.4.jar:git-Paper-192]
	at org.yaml.snakeyaml.composer.Composer.composeNode(Composer.java:144) ~[patched_1.14.4.jar:git-Paper-192]
	at org.yaml.snakeyaml.composer.Composer.composeValueNode(Composer.java:236) ~[patched_1.14.4.jar:git-Paper-192]
	at org.yaml.snakeyaml.composer.Composer.composeMappingChildren(Composer.java:227) ~[patched_1.14.4.jar:git-Paper-192]
	at org.yaml.snakeyaml.composer.Composer.composeMappingNode(Composer.java:215) ~[patched_1.14.4.jar:git-Paper-192]
	at org.yaml.snakeyaml.composer.Composer.composeNode(Composer.java:144) ~[patched_1.14.4.jar:git-Paper-192]
	at org.yaml.snakeyaml.composer.Composer.getNode(Composer.java:85) ~[patched_1.14.4.jar:git-Paper-192]
	at org.yaml.snakeyaml.composer.Composer.getSingleNode(Composer.java:108) ~[patched_1.14.4.jar:git-Paper-192]
	at org.yaml.snakeyaml.constructor.BaseConstructor.getSingleData(BaseConstructor.java:139) ~[patched_1.14.4.jar:git-Paper-192]
	at org.yaml.snakeyaml.Yaml.loadFromReader(Yaml.java:524) ~[patched_1.14.4.jar:git-Paper-192]
	at org.yaml.snakeyaml.Yaml.load(Yaml.java:452) ~[patched_1.14.4.jar:git-Paper-192]
	at org.bukkit.plugin.PluginDescriptionFile.<init>(PluginDescriptionFile.java:240) ~[patched_1.14.4.jar:git-Paper-192]
	at org.bukkit.plugin.java.JavaPluginLoader.getPluginDescription(JavaPluginLoader.java:164) ~[patched_1.14.4.jar:git-Paper-192]
	... 5 more
```

**Comment by mibby** [comment]
Here too.
https://github.com/RockinChaos/ItemJoin/commit/d9e26ef808e61f7073a2a5019da09c3de6673503#diff-5df3d3f42757c737a95531b868d1d2b9R127

**Comment by RockinChaos** [comment]
Whoops! Tested and seems to be loading now; http://ci.craftationgaming.com/job/ItemJoin/300/
Thank you for testing!

**Comment by RockinChaos** [comment]
This bug has been officially fixed in the release of ItemJoin v5.0.4.
You can grab it here; https://www.spigotmc.org/resources/itemjoin.12661/download?version=295178

Please read the changelog as the documentation has not been updated yet;
https://github.com/RockinChaos/ItemJoin/wiki/Recent-Changes
